Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "Flow.Launcher.Plugin.Everything": {
+      "commandName": "Project"
+    },
+    "Flow Launcher": {
+      "commandName": "Executable",
+      "executablePath": "D:\\Document\\Dotnet\\Flow.Launcher\\Output\\Release\\Flow.Launcher.exe"
```

Comment:
does this launchSettings.json file need to be checked in?

Reply:
Oh no, this should be ignored.